crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -337,6 +325,42 @@ impl Context {
         }
     }
 
+    /// Returns the a vector of `AcirVar`s constrained to be result of the function call.
+    ///
+    /// The function being called is required to be intrinsic.
```

Comment:
If this is only meant to be for intrinsics, then we should put that in the function name